### PR TITLE
Remove meta store from ray rdd

### PR DESCRIPTION
**Pull request by yushan111**
- [x]  to_spark_rdd and from_spark rdd 
- [x] assign partition to actors

### CODE REVIEW

File: pyzoo/zoo/orca/data/ray_rdd.py [hunk 1]
```diff
@@ -22,38 +23,47 @@
 from zoo.ray import RayContext
 
 
-@ray.remote(num_cpus=0)
-class MetaStore:
+# @ray.remote(num_cpus=0)
```
Comment (yangw1234) [inline pyzoo/zoo/orca/data/ray_rdd.py:26]: delete the code (not just comment it out)?

Comment (yushan111) [reply pyzoo/zoo/orca/data/ray_rdd.py]: Done

**Comment by yangw1234** [comment]
LGTM

**Comment by yushan111** [comment]
Jenkins passed: 
http://10.239.47.210:18888/view/ZOO-PR/job/ZOO-PR-Validation/4990/
http://10.239.47.210:18888/job/ZOO-PR-Python-Spark-2.4-py36k216/3354/
http://10.239.47.210:18888/job/ZOO-PR-Python-ExampleTests/2196/
http://10.239.47.210:18888/job/ZOO-PR-Python-ExampleTests-py37-ray-Spark3/397/

**Pull request merged**